qt/settingsdialog.py

```diff
@@ -1947,6 +1947,15 @@ def btnSnapshotsPathClicked(self):
 
                 self.config.removeProfileKey('snapshots.path.uuid')
 
+            fs = tools.filesystem(path)
+            if fs.startswith('ntfs'):
+                text = _('Warning: Destination filesystem for {path} is formatted with NTFS '
+                         'which has known incompatibilities with Unix style file systems.\n'
+                         'Are you sure you want to use it as your backup destination?')
```

Comment:
Beside avoiding escape characters like `\n` in the translation string I also try to avoid referencing user in person via "you".

```suggestion
                text = '\n'.join([
                       _('Warning: The destination filesystem for {path} is formatted with NTFS, '
                         'which has known incompatibilities with Unix-style filesystems.'),                             
                       _('Is this the backup destination to be used?')
                })
```